### PR TITLE
weighting for base values

### PR DESCRIPTION
**Pull request by matthewgalloway**

### CODE REVIEW

File: python-lib/dku_config/dku_model_handler.py [hunk 1]
```diff
@@ -53,21 +53,52 @@ def _compute_features(self):
...
-                self.base_values[feature] = feature_data.get('dropped_modality')
-                if self.base_values[feature] is None:
-                    # Handle the case where 'dropped_modality' is not available
-                    raise ValueError(f"Warning: 'dropped_modality' not found for feature {feature}. Please ensure drop one dummy is enabled and clipping uses max nb categories")
-            elif self.features[feature]['type'] == 'NUMERIC':
-                self.base_values[feature] = feature_data['stats']['average']
+            feature_type = self.features[feature]['type']
+
+            if feature_type == 'CATEGORY':
+                if exposure_column and train_df is not None:
```
Comment (david-behar) [inline python-lib/dku_config/dku_model_handler.py:67]: I don't understand what we are doing here. Are we not always taking the dropped modality as the base value?

File: python-lib/dku_config/dku_model_handler.py [hunk 2]
```diff
@@ -53,21 +53,52 @@ def _compute_features(self):
...
+                    else:
+                        self.base_values[feature] = 0  # Handle case where total exposure is 0
+                else:
+                    # Use .get to avoid KeyError if 'dropped_modality' key is missing
+                    self.base_values[feature] = feature_data.get('dropped_modality')
+                    if self.base_values[feature] is None:
+                        # Handle the case where 'dropped_modality' is not available
+                        raise ValueError(f"Warning: 'dropped_modality' not found for feature {feature}. Please ensure drop one dummy is enabled and clipping uses max nb categories")
+            elif feature_type == 'NUMERIC':
+                if exposure_column and train_df is not None:
```
Comment (david-behar) [inline python-lib/dku_config/dku_model_handler.py:88]: Weights can be input through the exposure or the weights, we should handle both cases

Comment (david-behar) [reply python-lib/dku_config/dku_model_handler.py:88]: I don't know if both can happen at the same time though

File: python-lib/dku_config/dku_model_handler.py (hunk 1 above)
Comment (matthewgalloway) [reply python-lib/dku_config/dku_model_handler.py:67]: We should do yes, lines 70-73 updated to reflect this

**Pull request merged**